packages/core/src/payment/strategies/stripev3/stripev3-error.ts

```diff
@@ -0,0 +1,25 @@
+import { StandardError } from '../../../common/error/errors';
+
+export enum StripeV3ErrorType {
+    AuthFailure = 'auth_failure',
+}
+
+export default class StripeV3Error extends StandardError {
```

Comment:
Do we need to add a new error class for this case? Can we leverage a previous one?

Reply:
I created the class so we can facilite for future the handling of stripe v3